recipes/cosmosis/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "cosmosis" %}
+{% set version = "0.5.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 305b8b09142886451ccd38abba3719a0ae104c1a33503a02abb6175af4d1755d
+
+build:
+  number: 0
+  skip: True  # [(py < 36) or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - compilers
+    - liblapack
+    - libblas
+    - {{ mpi }}
```

Comment:
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('fortran') }}
    - {{ compiler('cxx') }}```

Reply:
Thanks for fixing!  The MPI compiler is needed for polychord / multinest - should that not be in there?

Reply:
(Needed for full functionality; it will compile without)

Reply:
Ah the mpi compiler. I don't think just having {{ mpi }} in there will get that compiler. You'll want things like mpich-mpicc etc.

Reply:
It seemed to include them when I ran this locally. Or something included them.  You think there should be another package as well?